testsuite/module-publish/src/packages/simple/scripts/main.move

```diff
@@ -0,0 +1,17 @@
+script {
+    // Note: this constant can be replaced in compiled script to make it hash to a different value.
+    const SENDER: address = @0x1;
+
+    fun main(sender: &signer) {
+        // The idea is to to ensure that this script takes some time to be deserialized and verified, but the actual
```

Comment:
how about naming the package accordingly

Reply:
Is is part of `simple` package which contains a `simple` module, so either I need to rename that or create a new package just for script, which seems unnecessary?

Reply:
I see. Didn't realize it's an existing module. Is it even allowed to use a different file name or `fun` name then? `simple::main::main()` doesn't describe what you intend to do at all. But I guess it's no big deal.

Reply:
`main` is not a module here, but a script, can call it `script` ... hm.... `main` describes it as an entry point, but maybe we can rename either Move file, or function, good call